src/definition/livechat/IDepartment.ts

```diff
@@ -2,6 +2,13 @@ export interface IDepartment {
     id: string;
     name?: string;
     email?: string;
+    description?: string;
+    offlineMessageChannelName?: string;
+    requestTagBeforeClosingChat?: false;
+    chatClosingTags?: Array<string>;
+    abandonedRoomsCloseCustomMessage?: string;
+    waitingQueueMessage?: string;
+    departmentsAllowedToForward?: string;
```

Comment:
@renatobecker please review the data types and if these fields are really optional, as I don't have much familiarity with that domain 😛

Reply:
Everything looks good @d-gubert !
In fact, all of these fields are optional because, except description field, all the others are EE fields.